packages/indexer-generator/src/render-domain.ts

```diff
@@ -306,21 +326,16 @@ export default class MainDomain
     return { info, stats }
   }
 
-  async getAccountEventsByTime(
-    account: string,
-    startDate: number,
-    endDate: number,
-    opts: any,
-  ): Promise<StorageStream<string, ${Name}Event>> {
-    const stream = await this.context.apiClient
+  async getAccountEvents(args: EventsFilters): Promise<${Name}Event[]> {
+    const response = await this.context.apiClient
       .useBlockchain(BlockchainChain.Solana)
       .invokeDomainMethod({
-        account,
-        method: 'getAccountEventsByTime',
-        args: [startDate, endDate, opts],
+        account: args.account,
+        method: 'getAccountEvents',
+        args: [args],
```

Comment:
Ah, this reminds me of something: `args` only works well in Moleculer, if all the passed objects/values are primitive types. If `EventFilters` were to contain any fancy type like `BigNumber` or `BN` this would fuck things up on the other side.

Reply:
I think I should mention this somewhere in the docs, I guess. Maybe also add a comment here, warning the user to only pass primitive types to the `args` of `invokeDomainMethod()`